.github/workflows/prbot.yml

```diff
@@ -57,6 +57,16 @@ jobs:
       - uses: actions/checkout@v2
       - uses: ./.github/actions/setup
       - uses: ./.github/actions/e2e-atomic-screenshots
+  e2e-atomic-visual-tests:
+    name: 'Run Applitool visual regressions tests on Atomic'
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ./.github/actions/setup
+      - uses: ./.github/actions/e2e-visual-tests
+    env:
+      APPLITOOLS_API_KEY: ${{ secrets.APPLITOOLS_API_KEY }}
```

Comment:
This will need to be added into github

Reply:
You can share it with me on lastpass I will add it.